budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/persistence/person/PersonRepository.java

```diff
@@ -11,10 +11,10 @@
 
     List<PersonEntity> findByProjectIdOrderByNameAsc(long projectId);
 
-    @Query("select new org.wickedsource.budgeteer.persistence.person.PersonBaseDataBean(p.id, p.name, sum(r.minutes * r.dailyRate), sum(r.minutes), max(r.date)) from PersonEntity p left join p.workRecords r where p.project.id=:projectId group by p.id, p.name order by p.name")
+    @Query("select new org.wickedsource.budgeteer.persistence.person.PersonBaseDataBean(p.id, p.name, sum(r.minutes * r.dailyRate), sum(r.minutes), max(r.date), p.defaultDailyRate) from PersonEntity p left join p.workRecords r where p.project.id=:projectId group by p.id, p.name order by p.name")
```

Comment:
p.defaultDailyRate must be included into the group by clause as well to turn this into a valid query. See patch attached: 
[fix-missing-group-by.patch.txt](https://github.com/adessoAG/budgeteer/files/5486771/fix-missing-group-by.patch.txt)